content/docs/06-integrations/00-aws-ebs.md

```diff
@@ -32,6 +32,168 @@ AWS Elastic Block Store is an easy to use, high performance block storage at any
 </Tabs.TabPane>
 </Tabs>
 
+# Permissions
+
+For the below [Scenario](/integrations/aws-ebs#scenario:) the minimum iam [Policy](/integrations/aws-ebs#policytobecreated) described in this section needs to be provisioned to the AWS console:
+
+## Scenario:
+
+|AWS EBS Version | Kubernetes Version|
+|----------------|-------------------|
+| 1.15.x         | 1.22.x            |
+| 1.15.x         | 1.23.x            |
+| 1.18.x         | 1.22.x            |
+| 1.18.x         | 1.23.x            |
+
+## Policy to be created
+
+```{
```

Comment:
instead of this we can specify to use amazon managed policy `AmazonEBSCSIDriverPolicy`

Reply:
if user wishes to have encryption enabled 

{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "VisualEditor0",
            "Effect": "Allow",
            "Action": [
                "kms:GenerateDataKeyWithoutPlaintext",
                "kms:CreateGrant"
            ],
            "Resource": "*"
        }
    ]
}

use these permissions

Reply:
> if user wishes to have encryption enabled
> 
> { "Version": "2012-10-17", "Statement": [ { "Sid": "VisualEditor0", "Effect": "Allow", "Action": [ "kms:GenerateDataKeyWithoutPlaintext", "kms:CreateGrant" ], "Resource": "*" } ] }
> 
> use these permissions

added